### PR TITLE
[#557] Add FAQ page

### LINKED ISSUE #557: Add FAQ section
An FAQ section regarding the catalog should be added.
However, it is challenging to understand where such a section should be. Should we create a menu entry? If so, which menu?

### PR DESCRIPTION
**Pull request by libremente**
This PR:
- [x] Adds an FAQ page
- [x] Adds both the English and the Italian versions
- [x] Adds a YAML file to be used as FAQs DB
- [x] Add menu entry or explicit link to page - added in /software page, may be necessary to add somewhere else as well
- [x] Remove placeholder texts from faq.yml

Fixing:
* Fixes #557

### CODE REVIEW

File: _pages/it/software.md [hunk 1]
```diff
@@ -22,5 +22,6 @@ pair-blocks:
           url: '/it/search?type=software_open&page=0&sort=relevance'
 ---
 
-Questo è il catalogo previsto dalle [Linee Guida per l'Acquisizione e il Riuso del Software](/it/riuso), che include le soluzioni messe a riuso dalla Pubblica Amministrazione ai sensi dell'art. 69 e il software open source di terze parti destinato alla PA. 
-<a href="/it/riuso" class="btn btn-outline-primary mr-10 ml-auto mt-3">Maggiori informazioni su acquisizione e riuso</a>
+Questo è il catalogo previsto dalle [Linee Guida per l'Acquisizione e il Riuso del Software](/it/riuso), che include le soluzioni messe a riuso dalla Pubblica Amministrazione ai sensi dell'art. 69 e il software open source di terze parti destinato alla PA.   
+
+<a href="/it/riuso" class="btn btn-outline-primary mr-10 ml-auto mt-3">Maggiori informazioni</a> <a href="/it/faq" class="btn btn-primary mr-10 ml-auto mt-3 faq">FAQ</a>
```
Comment (libremente) [inline _pages/it/software.md:27]: If you approve this @sebbalex I can port it to the English page as well

File: _includes/faq.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+{% assign list = site.data.faq[page.lang] %}
+{% assign counter = 1 %}
+{% for item in list %}
```
Comment (libremente) [inline _includes/faq.html:4]: Should we add something in case there are no entries in the faq data file?

Comment (sebbalex) [reply _includes/faq.html:4]: yes, I found blank pages kind of misleading

Comment (libremente) [reply _includes/faq.html:4]: Good, gimme a sec

File: _pages/it/software.md (hunk 1 above)
Comment (sebbalex) [inline _pages/it/software.md:27]:
```suggestion
<a href="/it/riuso" class="btn btn-outline-primary mr-10 ml-auto mt-3">Maggiori informazioni</a>
<a href="/it/faq" class="btn btn-primary mr-10 ml-auto mt-3 faq">FAQ</a> 
```
🙏

Comment (libremente) [reply _pages/it/software.md:27]: Vim maxcol 80 makes great things happen :smile:

**Pull request merged**